packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -30,6 +30,7 @@ function getPaymentMethodTitle(
     const cdnPath = (path: string) => `${basePath}${path}`;
 
     return method => {
+        const paymentWithLogo = method.initializationData.methodsWithLogo !== undefined ? method.initializationData.methodsWithLogo : [];
```

Comment:
```suggestion
        const paymentWithLogo = method.initializationData?.methodsWithLogo ? method.initializationData.methodsWithLogo : [];
```
OR

```suggestion
        const paymentWithLogo = method.initializationData && method.initializationData.methodsWithLogo ? method.initializationData.methodsWithLogo : [];
```

Reply:
Hi Alan, this change is already do it, can you validate again the code, please?